verticapy/tests_new/performance/vertica/test_qprof.py

```diff
@@ -930,21 +930,20 @@ def test_get_qplan(self, qprof_data, return_report, print_plan):
         ],
     )
     @pytest.mark.parametrize(
-        "metric",
+        "metric, metric_info",
         [
-            "cost",
-            "rows",
-            "exec_time_ms",
-            "est_rows",
-            "proc_rows",
-            "prod_rows",
-            "rle_prod_rows",
-            "clock_time_us",
-            # "cstall_us",  # ZeroDivisionError
-            # "pstall_us",  # ZeroDivisionError
-            "mem_res_mb",
-            # "mem_all_mb",  # ZeroDivisionError
-            # ["cost", "proc_rows"],
+            ("cost", "Query plan cost"),
+            ("rows", "Estimated row count"),
+            ("exec_time_ms", "Execution time in ms"),
+            ("est_rows", "Estimated row count"),
+            ("proc_rows", "Processed row count"),
+            ("prod_rows", "Produced row count"),
+            ("rle_prod_rows", "Produced RLE row count"),
+            ("clock_time_us", "Clock time in us"),
```

Comment:
same microseconds

Reply:
don't think it is necessary otherwise we will have to do it for everything